### DISCUSSION TITLE
How to check which server version is installed

### DISCUSSION THREAD

**Discussion by JappeHallunken** [dani-garcia/vaultwarden#4321] [Q&A] [answered]
I'm running vaultvarden without docker, just the plain binary.
In the web interface I see just the version of the web interface, as expected.
When I run `vaultwarden --version` I just get `Version info from git not present`.

When I check the logs via `journalctl` there is also no hint about the version.

How can I check which version I'm currently running?

**Comment by BlackDex** [comment]
Then it's a bit difficult. Because the version info is fetched during build-time.
If for some reason the binary you have is build without that information, there is also no good way for us to tell.

**Comment by stefan0xC** [comment, marked as answer]
If you check the `/api/config` endpoint it should give you an idea if the version is way too old (because we've hard-coded the `version` to `2023.9.1` about 4 months ago): https://github.com/dani-garcia/vaultwarden/commit/cb4b683dcd51eff4508bcf50e34d657b8d2225d4#diff-0c15871c114078d2cdd4a58dc463a77a055ab8ae4ef41583685afcc4aa062e45

**Comment by JappeHallunken** [comment]
Thx, I checked `/api/config` and I've got `2023.9.1`

Edit: I also recognized your typo, so I guess I'm on a pretty recent version.
